### ISSUE TITLE
Company-tng doesn't work with company-box

### ISSUE THREAD

**Issue by mssdvd** [sebastiencs/company-box#47]
I can't use `company-tng` with `company-box` because the candidate isn't displayed in the buffer so I can't select it.
![image](https://user-images.githubusercontent.com/18464329/42751544-67ec25f0-88eb-11e8-95ea-4390761bb6db.png)

This is my config:
```elisp
(use-package company
  :defer 1
  :config
  (company-tng-configure-default)
  (global-company-mode))

(use-package company-box
  :hook (company-mode . company-box-mode))
```
This is the value of `company-frontends`:
`(company-tng-frontend company-pseudo-tooltip-frontend company-echo-metadata-frontend)`

**Comment by jmorag** [comment]
I can confirm that company-tng is broken for me as well with company-box enabled.

**Comment by t-hart** [comment]
Same here. I did manage to get it to work using `tab` following the advice in [this reddit thread](https://www.reddit.com/r/emacs/comments/8h9fdg/a_company_frontend_with_icons/), but it failed the second time I tried. More importantly, it only worked with `tab` and not with `M-n/M-p`, which I normally use for cycling. Not sure exactly what needs to be done to enable this, but would love to see these two come together somehow.
